module/observations/helper.js

```diff
@@ -2255,6 +2253,48 @@ module.exports = class ObservationsHelper {
         })
     }
 
+
+ /**
+      * Get imported observation documents.
+      * @method
+      * @name getStartedObservations
+      * @param  {String} userId - userId of user.
+      * @param  {String} programId - program Id.
+      * @returns {result} - all the observation which user has started in that program. 
+     */
+
+    static getStartedObservations(userId, programId){
```

Comment:
@ankitpws this will be moved to usersHelper

Reply:
Moved under userHelper